src/api/createDeviceReferenceData/tests/test_index.py

```diff
@@ -71,6 +72,7 @@ def test_index() -> None:
                 "pathParameters": {
                     "product_team_id": str(product.product_team_id),
                     "product_id": str(product.id),
+                    "env": Environment.DEV,
```

Comment:
better to name `environment` rather than `env`? `env` to me sounds like environment variables

Reply:
yeah, probably.